nl_preprocessing.py

```diff
@@ -84,17 +84,14 @@ def remove_punctuation_and_linebreaks(sentence: str) -> str:
         """
         Removes and replaces non-textual elements
 
-        Removes whitespace and all punctuation except questions marks. Question marks are replaced with full stops,
-        as we do not want to care about questions. Full stops not followed by a space are replace with a space, e.g.
-        object.property -> object property.
+        Removes whitespace and all punctuation except questions marks. Question marks and full stops are replaced with
```

Comment:
Could you update the docstring from: `Removes whitespace and all punctuation except questions marks` to `Removes whitespace and all punctuation.` since questions marks are removed too.